carbon-design-system/carbon-addons-iot-react
# feat(listitem): add tags

Closes #1518 

**Summary**
Adds the possibility to add one or more tags to an ListItem in a List.
![image](https://user-images.githubusercontent.com/5167091/90565440-37317e80-e1a7-11ea-842f-8dc1ae3753cb.png)
![image](https://user-images.githubusercontent.com/5167091/90565487-44e70400-e1a7-11ea-9197-14283738704f.png)


**Change List (commits, features, bugs, etc)**
1. Added a new tag prop to the ListItem.jsx and a new ListTagsPropTypes PropType in SharedPropTypes.js since the prop type is shared between List and ListItem
2. Updated ListItem tests and stories
3. Modified the content prop in List.jsx to include an optional tags prop
4. Updated List tests and stories

**Acceptance Test (how to verify the PR)**

1. Go to /story/watson-iot-experimental-listitem--basic-w-knobs
2. Enable tags using the knobs, verify that it looks ok
3. Enable all the other features and make sure it all plays well together
4. Verify that the rest of the ListItem stories look good
5. Go to /story/watson-iot-experimental-list--with-tags
6. Make sure that the tag is rendered properly for each item
7. Verify that the rest of the List stories look good
